website/src/components/expandable/index.js

```diff
@@ -39,43 +39,46 @@ function Expandable({ children, alt_header = null, lifecycle }) {
     popup.classList.add('copy-popup');
     popup.innerText = 'Link copied!';
 
-  // Add close button ('x')
-  const closeButton = document.createElement('span');
-  closeButton.classList.add('close-button');
-  closeButton.innerHTML = ' &times;'; // '×' symbol for 'x'
-  closeButton.addEventListener('click', () => {
-    if (document.body.contains(popup)) {
-      document.body.removeChild(popup);
-    }
-  });
-  popup.appendChild(closeButton);
+    // Add close button ('x')
+    const closeButton = document.createElement('span');
+    closeButton.classList.add('close-button');
+    closeButton.innerHTML = ' &times;'; // '×' symbol for 'x'
+    closeButton.addEventListener('click', () => {
+      if (document.body.contains(popup)) {
+        document.body.removeChild(popup);
+      }
+    });
+    popup.appendChild(closeButton);
 
-  document.body.appendChild(popup);
+    document.body.appendChild(popup);
 
-  setTimeout(() => {
-    if (document.body.contains(popup)) {
-      document.body.removeChild(popup);
-    }
-  }, 3000);
-};
+    setTimeout(() => {
+      if (document.body.contains(popup)) {
+        document.body.removeChild(popup);
+      }
+    }, 3000);
+  };
 
-useEffect(() => {
-  if (window.location.hash === `#${anchorId}`) {
-    setOn(true);
-    const element = document.getElementById(anchorId);
-    if (element) {
-      element.scrollIntoView({ behavior: 'smooth' });
+  useEffect(() => {
+    if (window.location.hash === `#${anchorId}`) {
+      setOn(true);
+      const element = document.getElementById(anchorId);
+      if (element) {
+        element.scrollIntoView({ behavior: 'smooth' });
+      }
     }
-  }
-}, [anchorId]);
+  }, [anchorId]);
 
   return (
-    <div id={anchorId} className={`${styles.expandableContainer} `}>
+    <div id={anchorId} className={`${styles.expandableContainer}`}>
       <div className={styles.header} onClick={handleToggleClick}>
         <span className={`${styles.toggle} ${isOn ? styles.toggleDown : styles.toggleRight}`}></span>
         &nbsp;
         <span className={styles.headerText}>
-          {alt_header}<Lifecycle status={lifecycle} />
+          {alt_header}
+          <span onClick={(e) => e.stopPropagation()}>
```

Comment:
this stops the copy link propagation so that users can click the pill for the link or click on the icon to copy the url